### PR TITLE
Done - Snackbar

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**
Snackbar Completed

### CODE REVIEW

**Review by MayankKumar10 — COMMENTED** [review]
I have seen inline styling avoid using that else everything LGTM

**Comment by MuskanChhatrasal** [comment]
Okay will follow that. Thankyou

File: components/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.snackbar {
+  background-color: #374151;
```
Comment (harshkotkar5557) [inline components/snackbar.css:2]: You use variable for color

**Review by harshkotkar5557 — COMMENTED** [review]
Looks to me. you have follow almost every point of guideline.

**Review by vaibhav18matere — COMMENTED** [review]
looks fair to proceed

**Pull request merged**